create documentation for operations using the new Beagle dsl (#767)
ZupIT/beagle-docs: content/pt/resources/dsl-operations/comparison-operator.md

---
title: Operadores de comparação
weight: 367

> description: 'Nesta seção, você encontra descrição completa dos Operadores de Comparação.'

Reviewer: ```suggestion
description: 'Nesta seção, você encontra descrição completa dos operadores de comparação.'
```

---

---

## Quais são? 

> 

Reviewer: Veja abaixo quais são os operadores de comparação:

        valor for <strong>maior ou igual</strong> ao segundo.</td>
    </tr>
    <tr>
      <td style="text-align:center">
        lt
        (Menor)
      </td>
      <td style="text-align:left">lt(constant(3), constant(4))</td>
      <td style="text-align:left">Operador que recebe duas entradas e resulta em <strong>verdadeiro</strong> se o primeiro
        valor for <strong>menor</strong> que o segundo.</td>
    </tr>
    <tr>
      <td style="text-align:center">
        lte
        (Menor ou igual)
      </td>
      <td style="text-align:left">lte(constant(3), constant(4))</td>
      <td style="text-align:left">Operador que recebe duas entradas e resulta em <strong>verdadeiro</strong> se o primeiro
        valor for <strong>menor ou igual</strong> ao segundo.</td>
    </tr>
    <tr>
      <td style="text-align:center">
        eq
        (Igual)
      </td>
      <td style="text-align:left">eq(constant(4), constant(4))</td>
      <td style="text-align:left">Operador que recebe duas entradas e resulta em <strong>verdadeiro</strong> se o os
        dois valores s&#xE3;o <strong>iguais</strong>.</td>
    </tr>
  </tbody>
</table>

### Exemplo

> Aqui, o exemplo é de uma tela que utiliza a operação de comparação **lt**, que verifica se o valor da soma de counter e:

Reviewer: ```suggestion
O exemplo aqui é de uma tela que utiliza a operação de comparação **lt**, ela verifica se o valor da soma de counter e há duas possibilidades:
```

> * Se o resultado da condição do `or` for verdadeira, o texto do componente `Text` se torna **true;**
> * Caso falso ele atribui ao texto o valor de **false.**

Reviewer: ```suggestion
1. Se o resultado da condição do `**or**` for verdadeira, o texto do componente `Text` se torna **true**;
2. Se for falso, ele atribui ao texto o valor de **false.**

Veja abaixo:
```
